src/status_im/common/check_before_syncing/style.cljs

```diff
@@ -0,0 +1,18 @@
+(ns status-im.common.check-before-syncing.style)
+
+(def buttons-container
+  {:flex-direction   :row
+   :justify-content  :space-between
+   :gap              12
+   :padding-vertical 12})
+
+(def button {:flex 1})
+
+(def checkbox {:margin-top 16 :flex-direction :row :align-items :center})
+(def checkbox-text {:margin-left 10})
```

Comment:
Thank you @yqrashawn for your PR.
Please format these lines. (empty lines and map)